project-base/tests/App/Acceptance/acceptance/CartCest.php

```diff
@@ -307,4 +293,23 @@ public function testPromoCodeFlowInCart(
         $cartPage->canSeePromoCodeSubmitButtonElement();
         $cartPage->assertTotalPriceWithVatByProducts($products);
     }
+
+    /**
+     * @param \Tests\App\Test\Codeception\AcceptanceTester $me
+     * @param string $productName
+     * @param int $quantity
+     */
+    protected function seeRightMessageForAddedProducts(AcceptanceTester $me, string $productName, int $quantity): void
```

Comment:
Doesn't this method make sense in CartPage more?
That way it can be reused in another test

Reply:
Done